### PR TITLE
Fixes #3264 :right sidebar not shown on sessions and schedule tab

### LINKED ISSUE #3264: Public Schedule Page/Session Page: Do not show right sidebar
The right sidebar on the schedule pages reduces the space for the actual schedule.

Please take out the right sidebar on the following pages and expand the schedule/session area to make use of the additional space:
* [x] the public schedule page (and subpages)
* [x] the public session page

screenshot from 2017-02-26 19-05-12

### PR DESCRIPTION
**Pull request by Achint08**
Fixes #3264 

Schedule page
screenshot from 2017-02-27 01 59 23

Sessions page

screenshot from 2017-02-27 02 00 07

Other pages (e.g. speakers)

screenshot from 2017-02-27 02 00 24

Kindly review :)

### CODE REVIEW

File: app/templates/gentelella/guest/event/base.html [hunk 1]
```diff
@@ -156,13 +156,14 @@ <h4 style="font-weight: 300;"><i class="glyphicon glyphicon-map-marker"></i> {{
         </div>
     </div>
     <div class="row">
-        <div class="col-md-8 col-sm-8 col-xs-12 col-md-push-1 col-sm-push-2 event-info-container-holder"
+        <div class="col-xs-12 col-md-push-1 col-sm-push-2 event-info-container-holder {% if active_page =='sessions' or active_page == 'schedule' %}col-md-10 col-sm-10 {% else %}col-md-8 col-sm-8{% endif %}" 
              style="min-height: 600px;">
             {% block content %}
             {% endblock %}
         </div>
         <div class="col-md-2 col-md-push-1 hidden-xs hidden-sm">
```
Comment (niranjan94) [inline app/templates/gentelella/guest/event/base.html]: Put the if statement around this entire div

File: app/templates/gentelella/guest/event/base.html [hunk 2]
```diff
@@ -156,13 +156,14 @@ <h4 style="font-weight: 300;"><i class="glyphicon glyphicon-map-marker"></i> {{
         </div>
     </div>
     <div class="row">
-        <div class="col-md-8 col-sm-8 col-xs-12 col-md-push-1 col-sm-push-2 event-info-container-holder"
+        <div class="col-xs-12 col-md-push-1 col-sm-push-2 event-info-container-holder {% if active_page =='sessions' or active_page == 'schedule' %}col-md-10 col-sm-10 {% else %}col-md-8 col-sm-8{% endif %}"
```
Comment (niranjan94) [inline app/templates/gentelella/guest/event/base.html]: And keep the length of a line very long. Try to wrap the single line into multiple lines. makes it more readable

File: app/templates/gentelella/guest/event/base.html (hunk 1 above)
Comment (Achint08) [inline app/templates/gentelella/guest/event/base.html]: Hello @niranjan94 I tried what you wanted. Is this what you asked for?

File: app/templates/gentelella/guest/event/base.html [hunk 3]
```diff
@@ -156,13 +156,19 @@ <h4 style="font-weight: 300;"><i class="glyphicon glyphicon-map-marker"></i> {{
...
+        <div class="col-md-10 col-sm-10 col-xs-12 col-md-push-1 col-sm-push-2 event-info-container-holder" 
+        style="min-height: 600px;">
+         {% else %}
+         <div class = "col-md-8 col-sm-8 col-xs-12 col-md-push-1 col-sm-push-2 event-info-container-holder" 
+         style="min-height: 600px;">
+         {% endif %}" 
             {% block content %}
             {% endblock %}
         </div>
         <div class="col-md-2 col-md-push-1 hidden-xs hidden-sm">
```
Comment (niranjan94) [inline app/templates/gentelella/guest/event/base.html]: Let this div be inside the if statement instead of the if statement being inside this div.

Comment (Achint08) [inline app/templates/gentelella/guest/event/base.html]: Got it. Sorry. My bad.

**Comment by Achint08** [comment]
@niranjan94 Done now. Thanks for guiding me. :)

**Pull request merged**